node/_bin/munin-asyncd.in

```diff
@@ -143,7 +143,11 @@ my $last_cleanup=0;
 MAIN: while($keepgoing) {
 	my $when = time;
 
-	my $when_next = $when + $timeout; # wake up at least every $timeout sec
+	my $when_next = int($when / $minrate) * $minrate + 180;
```

Comment:
How about the following instead? (without hard-coding a value independent of `$minrate`)
```perl
# start the next run close to the end of a munin-node update operation (i.e. try to avoid overlapping activities)
my $when_next = int((int($when / $minrate) + 0.75) * $minrate);
```

Reply:
Yes, it should use 0.75 * $minrate instead of 180.